README.md

```diff
@@ -1,41 +1,41 @@
 # Atleta Network
 
-The project is built on a [frontier template](https://github.com/PureStake/frontier) —
-[FRAME](https://docs.substrate.io/v3/runtime/frame)-based
-[Substrate](https://substrate.io) node with the Ethereum RPC support, ready for
+The project is built on a [frontier template](https://github.com/polkadot-evm/frontier) —
+[FRAME](https://docs.substrate.io/reference/)-based
+[Substrate](https://substrate.io) node with the [Ethereum RPC](https://ethereum.org/en/developers/docs/apis/json-rpc/#json-rpc-methods) support, ready for
 hacking :rocket:
 
 
 
 
 ## Build & Run
 
-#### Build with "fast-runtime" feature
-To build the test/dev network with the "fast-runtime" feature, execute the following command from the project root:
-```
-$ cargo build --release --features "fast-runtime" 
+#### Build with `fast-runtime` feature
```

Comment:
Probably, we should just add a section with the list of the features and describe them?

Reply:
Yep, done.